en/features_mc/throw_launch.md

```diff
@@ -0,0 +1,92 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios. various platforms and in different scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+The vehicle detects that it has been thrown based on reaching a certain speed (5m/s), and then starts the motors at the apex of the throw, once it determines that it has started to fall.
+You need to throw it high enough so that the vehicle can stabilize its height well before falling in range of people (2 or 3 metres is usually sufficient).
+
+Note that unlike in other configurations, when throw launch is enabled the propellers should not spin until the launch is detected (even when the vehicle is armed), and that the arming tone will continue to play until the vehicle is thrown.
+The vehicle will not automatically disarm after arming, and must be manually disarmed if you choose not to throw it. 
+
+The vehicle should not be transported after being armed and before the throw.
+In particular, the throw should not be executed from a moving platform.
+The reason for this is that the condition to start the motors depends on absolute speed of the multicopter and does not account for any additional movement.
+Trying to throw the drone from a moving platform might result in the motors being started prematurely.
+
+## Safety
+
+:::warning
+Throw launch is potentially dangerous as it requires the operator to hold an armed multicopter and be in proximity when it is flying.
```

Comment:
```suggestion
Throw launch is dangerous as it requires the operator to hold an armed multicopter and be in proximity when it is flying.
```
Let's be honest.

Reply:
I've played with a throwable caged rotor copter at about 200g that wasn't scary. But I take your point.

Reply:
I destroyed the keyboard of my laptop using a 35g micro quad. I agree it's less dangerous but if my finger (or eye) would have taken that hit that would have lead to injury...

Reply:
Ouch!